tests/Issue3278.py

```diff
@@ -0,0 +1,37 @@
+############################ Copyrights and license ############################
+#                                                                              #
+# Copyright 2025 Zachary Haberman <6599715+interifter@users.noreply.github.com>#
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.readthedocs.io/                                              #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+################################################################################
+
+from . import Framework
+
+
+class Organization(Framework.TestCase):
+    def testIssue3278CreateTeam(self):
+        super().setUp()
+        team_name = "team-name"
+        user_name = "interifter"
+        self.user = self.g.get_user(user_name)
+        self.org = self.g.get_organization("testinterifter")
+        team = self.org.create_team(team_name, maintainers=[user_name])
+        team = self.org.get_teams()[0]
+
+        self.assertEqual(team_name, team.name)
+        self.assertEqual(user_name, team.get_members(role="maintainer")[0].name)
```

Comment:
Adding this assertion to `Organization.testCreateTeamWithAllArguments` would be great
```
        self.assertEqual(maintainer.login, team.get_members(role="maintainer")[0].name)
```
Simply add (and adjust) the last request of `tests/ReplayData/Organization.testIssue3278CreateTeam.txt` (https://github.com/PyGithub/PyGithub/pull/3291/files#diff-079f3aa92b2a83a936f88bd3d8fbc6997361881b1c58b1a62b539282a142e71eR45-R54) to `tests/ReplayData/Organization.testCreateTeamWithAllArguments.txt`.

Reply:
I believe I've addressed this